website/docs/features/cors.mdx

```diff
@@ -0,0 +1,85 @@
+---
+id: cors
+title: CORS with Yoga
+sidebar_label: CORS
+---
+
+Yoga comes with CORS support out of the box - CORS can be configured when creating the server either by passing a CORSOptions object, or a builder function that returns the CORSOptions object.
```

Comment:
Maybe it makes sense to add 1 or two sentences of what CORS is and why its important?

Reply:
yup gotcha - we usually do in Redwood - but thought your audience might be a little different. Copy paste time!

Reply:
Done ✅